package.json

```diff
@@ -297,7 +297,7 @@
     "vscode-codicons": "^0.0.4"
   },
   "devDependencies": {
-    "@types/istanbul-lib-coverage": "^2.0.2",
+    "@types/istanbul-lib-coverage": "^2.0.3",
```

Comment:
See comment below about FileCoverage. This change was part of my attempt to address the issue. It didn't help.